src/storage/src/table/mem_table.rs

```diff
@@ -24,8 +23,23 @@ use crate::error::StorageResult;
 pub enum RowOp {
     Insert(Row),
     Delete(Row),
-    Update((Row, Row)),
+    Update(Row),
 }
+impl RowOp {
+    pub fn is_insert(&self) -> bool {
+        matches!(self, Self::Insert(_))
+    }
+
+    pub fn is_delete(&self) -> bool {
+        matches!(self, Self::Delete(_))
+    }
+
+    pub fn is_update(&self) -> bool {
+        matches!(self, Self::Delete(_))
+    }
+}
```

Comment:
should we maintain the is_insert and is_update method? now our enum has express them clearly and the wrapping seems not needed.